src/word_debt_bot/game/core.py

```diff
@@ -55,3 +55,25 @@ def add_debt(self, player_id: str, amount: int):
         state = self._state
         state[player_id].word_debt += amount
         self._state = state
+
+    def create_leaderboard(self, sort_by: str, lb_len: int):
+        if sort_by not in ["debt", "cranes"]:
```

Comment:
Usability: It might be better UX if we `.lower()` here to avoid unnecessary errors, `.strip()` might also be helpful but I believe the command interface handles that automatically.

Reply:
`.lower()` added prior to the if branches for the sorting key.

Testing for extra whitespace from users showed that the command interface appears to be handling striping of it:
```
>.leaderboard      debt       10
1. User1 - 10,000 debt - 0 cranes
. . .
10. User10 ...
```

However, an input like: `.leaderboard   d ebt   3` causes a ValueError to be logged in discord.log

->Will work on a way to properly display error to user.